map/lanelet2_extension/CMakeLists.txt

```diff
@@ -6,6 +6,7 @@ if(NOT CMAKE_CXX_STANDARD)
 endif()
 
 find_package(ament_cmake_auto REQUIRED)
+find_package(Eigen3 REQUIRED)
```

Comment:
Since Eigen3 is linked by underliying lanelet_core package. you shouldn't need this change.

Reply:
@mitsudome-r this is fixed